### ISSUE TITLE
Batch normalization could not be transformed?

### ISSUE THREAD

**Issue by rbotafogo** [zzh8829/yolov3-tf2#69]
Just installed your library and its working but giving the following warnings:

WARNING:tensorflow:Entity <bound method BatchNormalization.call of <yolov3_tf2.batch_norm.BatchNormalization object at 0x00000229F1365A08>> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: Bad argument number for Name: 3, expecting 4
W0918 22:49:38.362240 30700 ag_logging.py:146] Entity <bound method BatchNormalization.call of <yolov3_tf2.batch_norm.BatchNormalization object at 0x00000229F1365A08>> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <yolov3_tf2.batch_norm.BatchNormalization object at 0x00000229F13FF6C8>> could not be transformed and will be executed as-is. Please report this to the AutoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: Bad argument number for Name: 3, expecting 4

**Comment by rbotafogo** [comment]
switching to gast 0.2 version seems work
